plugins/orchestrator-backend/src/service/SonataFlowService.ts

```diff
@@ -273,7 +273,22 @@ export class SonataFlowService {
 
       if (response.ok) {
         const json = await response.json();
-        return json.data.ProcessInstances;
+        const processInstancesSrc = json.data
+          .ProcessInstances as ProcessInstance[];
+        const workflowItems = await this.fetchWorkflows();
```

Comment:
(just FYI) Soon workflow definitions will be fetched from the Data Index. Hence, I think we can fetch all the information needed in this function in one shot.

Reply:
Can we refactor once this is done?

Reply:
Absolutely!